web/docs/project/env-vars.md

```diff
@@ -85,9 +85,10 @@ In the root of your Wasp project you can create two distinct files:
     REACT_APP_SOME_VAR_NAME=somevalue
     ```
 
-These files should not be committed to version control, and they are already ignored by default in the `.gitignore` file that comes with Wasp.
+`.env.server` should not be committed to version control as it can contain secrets and is therefore by default already ignored in the `.gitignore` file that comes with a new Wasp app.
+On the other hand, `.env.client` can be commited as it must not contain any secrets.
 
-<!-- `dotenv` files are a popular method for storing configuration: to learn more about them in general, check out the [README of the lib we use for them](https://github.com/stackbuilders/dotenv-hs). -->
```

Comment:
This text is inside of a HTML comment which means it won't be visible in the docs. Is this what we want?

Reply:
Good question I haven't noticed that! I guess somebody commented it out. I think I will comment it "in" back to life.

Reply:
Ok, took care of this! I uncommented it + put it into "info" block.